lib/huginn_acumen_product_agent/concerns/acumen_product_query_concern.rb

```diff
@@ -166,6 +166,7 @@ def parse_product_request(products)
                 variant['bookFormat'] = "http://schema.org/EBook"
                 variant['accessMode'] = "textual"
                 variant['isDigital'] = true
+                variant['isDefault'] = true
```

Comment:
I'm not sure I'm following the reason for this. This will flag every ebook as `isDefault`. This looks like a big commerce specific work-around, which needs to happen on the big commerce side.